playbooks/roles/kibana/README.md

```diff
@@ -3,6 +3,10 @@ Kibana
 
 A role to install Kibana which allows us to view logs stored in Elasticsearch.
 
+Important Note
+--------------
+If ELK stack already installed in the server and running this has an issue with Kibana not being able to connect with Elasticsearch then check README on the ``elasticsearch`` Role. You might need to set up the Kibana user's password manually.
```

Comment:
Can we link the steps required for enabling alerting on kibana for some context on why we need all these options and more details on setting them up? Mostly the docs here: https://www.elastic.co/guide/en/kibana/7.x/alert-action-settings-kb.html

Reply:
I've linked the suggested doc with additional information.